README.md

```diff
@@ -47,16 +47,18 @@ The following workflow is for a new **Helm** operator:
 
 ## Quick Start
 
-First, checkout and install the operator-sdk CLI:
+### Install the Operator SDK CLI
+
+#### Linux
+
+```sh
+curl -Lo operator-sdk https://github.com/operator-framework/operator-sdk/releases/download/v0.7.0/operator-sdk-v0.7.0-x86_64-linux-gnu && chmod +x operator-sdk && sudo cp operator-sdk /usr/local/bin/ && rm operatr-sdk
```

Comment:
@shawn-hurley Agreed, we already have a good guide in https://github.com/operator-framework/operator-sdk/blob/51d28df214440d97031368d7d700a2805161015d/doc/dev/release.md#verifying-a-release should I make a note and point to that?

Reply:
Maybe we should move that section to this new user-facing guide and link to it from our dev doc?

Reply:
I think when doing a release I would prefer to have all the steps in one document and not jump around, but happy to move it here if the majority thinks otherwise or duplicate it.

Reply:
I'm just thinking that from the user perspective, it might be a little weird to end up in a doc that's otherwise meant for the maintainers.

Reply:
@joelanford @shawn-hurley Done, PTAL.